src/module_2/groceries_analysis.md

```diff
@@ -0,0 +1,1811 @@
+```python
+%reset -f
+```
+
+# Importing data
+
+Data has been stored in local using aws cli. 
+
+
+```python
+import matplotlib.pyplot as plt
+import numpy as np
+import os
+import pandas as pd
+import seaborn as sns
+
+from IPython.display import Markdown, display
+
+DATA_PATH = "/home/alejandro/Zrive"
+
+abandoned_carts_df = pd.read_parquet(os.path.join(DATA_PATH, 'abandoned_carts.parquet'))
+inventory_df = pd.read_parquet(os.path.join(DATA_PATH, 'inventory.parquet'))
+orders_df = pd.read_parquet(os.path.join(DATA_PATH, 'orders.parquet'))
+regulars_df = pd.read_parquet(os.path.join(DATA_PATH, 'regulars.parquet'))
+users_df = pd.read_parquet(os.path.join(DATA_PATH, 'users.parquet'))
+
+feature_frame = pd.read_csv(os.path.join(DATA_PATH, 'feature_frame.csv'))
+
+
+def printmd(string):
+    display(Markdown(string))
+```
+
+# Part 1
```

Comment:
Awesome that you give context for the reader. Nice good practice. I could only add to incorporate a name to "Part 1" so we know what Part 1 is going to be about